src/main/java/commands/DeleteCommand.java

```diff
@@ -0,0 +1,66 @@
+package commands;
+
+import exceptions.MorganException;
+import storage.Storage;
+import tasks.Task;
+import tasks.TaskList;
+
+/**
+ * This is an DeleteCommand Class, which inherits from Command.
+ * The execution of this command will delete a task from the task list.
+ */
+public class DeleteCommand extends Command {
+    public static final String KEYWORD = "delete";
+    private static final String INPUT_FORMAT = String.format("\t\"%s [task number]\"", KEYWORD);
+    private static final String INPUT_FORMAT_ERROR = String.format("Please "
+            + "ensure your input is in the following format:\n" + INPUT_FORMAT);
+    private static final String TASK_NUMBER_ERROR = String.format("Please "
+            + "provide a valid task number.");
+    private final int taskNumber;
+
+    /**
+     * Constructor for DeleteCommand
+     * @param userInput The input string entered by the user.
+     * @throws MorganException If input format is invalid.
+     */
+    public DeleteCommand(String userInput) throws MorganException {
+        String intString = userInput.substring(KEYWORD.length()).trim();
+
+        // Checks if user specified task number
+        if (intString.isEmpty()) {
```

Comment:
Perhaps you could consider throwing this statement inside the try block, and have a catch block for the MorganException as well?